codersdk/agentsdk/agentsdk.go

```diff
@@ -567,20 +567,21 @@ func (c *Client) PostStartup(ctx context.Context, req PostStartupRequest) error
 	return nil
 }
 
-type StartupLog struct {
+type Log struct {
 	CreatedAt time.Time         `json:"created_at"`
 	Output    string            `json:"output"`
 	Level     codersdk.LogLevel `json:"level"`
+	Stage     string            `json:"stage"`
```

Comment:
Should we match the db here and call this source?

Reply:
100%